crates/cli/src/main.rs

```diff
@@ -209,27 +208,42 @@ fn init_db(command: &ContenderSubcommand) -> Result<(), CliError> {
 }
 
 fn init_tracing() {
-    let filter = EnvFilter::try_from_default_env().ok(); // fallback if RUST_LOG is unset
-    #[cfg(feature = "async-tracing")]
```

Comment:
we still need the `async-tracing` conditional, looks like it was just removed...